crates/api_models/src/enums.rs

```diff
@@ -220,6 +221,7 @@ pub enum RoutableConnectors {
     Payme,
     Paypal,
     Payu,
+    Plaid,
```

Comment:
Please remove this. The only connectors that belong in the `RoutableConnectors` enum are the ones merchants can use for making payments.

Reply:
I added it because It was being mapped in Kgraph_utils enum. Since `IntoDirValue` mapping itself is not needed now, i'll remove Plaid from RoutableConnectors